Errors finding libccd when building on Windows Subsystem for Linux

When trying to build on Windows Subsytem for Linux I am getting some weird errors. With `catkin_make_isolated`, I am getting this error.

`
from /home/wasonj/catkin_ws/src/tesseract/tesseract/tesseract_collision/src/fcl/fcl_utils.cpp:56:
/home/wasonj/catkin_ws/devel_isolated/fcl/include/fcl/narrowphase/detail/convexity_based_algorithm/gjk_libccd.h:41:10: fatal error: ccd/ccd.h: No such file or directory
 #include <ccd/ccd.h>
          ^~~~~~~~~~~
`

With `catkin build`, I am getting this error:

`
Errors     << tesseract_collision:make /home/wasonj/catkin_ws/logs/tesseract_collision/build.make.000.log
/usr/bin/ld: cannot find -lccd
collect2: error: ld returned 1 exit status
make[2]: *** [libtesseract_collision_fcl.so] Error 1
make[1]: *** [CMakeFiles/tesseract_collision_fcl.dir/all] Error 2
make: *** [all] Error 2
`

It appears something is wrong with how libccd is referenced.

--- a/tesseract/tesseract_collision/CMakeLists.txt
+++ b/tesseract/tesseract_collision/CMakeLists.txt
@@ -10,6 +10,9 @@ find_package(tesseract_common REQUIRED)
 find_package(Bullet REQUIRED)
 find_package(fcl REQUIRED)
 
+# We need to find ccd ourselves because FCL doesn't do it in their fclConfig.cmake
+find_package(ccd REQUIRED)
+
 # Create interface for core
 add_library(${PROJECT_NAME}_core INTERFACE)
 target_link_libraries(${PROJECT_NAME}_core INTERFACE tesseract::tesseract_geometry ${Boost_LIBRARIES} ${BULLET_LIBRARIES})

--- a/tesseract/tesseract_collision/cmake/tesseract_collision-config.cmake.in
+++ b/tesseract/tesseract_collision/cmake/tesseract_collision-config.cmake.in
@@ -17,5 +17,6 @@ find_dependency(tesseract_geometry)
 find_dependency(tesseract_common)
 find_dependency(Bullet)
 find_dependency(fcl)
+find_dependency(ccd)
 
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")

